tracing/tracer.go

```diff
@@ -39,22 +40,27 @@ var Configured bool
 type Config struct {
 	Jaeger      Jaeger
 	Stackdriver Stackdriver
+	Lightstep   Lightstep
 }
```

Comment:
let's add a `ServiceName` field here (and promote the other fields to go-flags groups) which will be used to define a service name attribute for the `trace.Provider` regardless of the exporter. Let's also have it define a default for `Jaeger.Service`.